test/fixtures/collect_series_fixtures.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+
+@pytest.fixture
+def season():
```

Comment:
just curious why we do this instead of just putting this right into the test?

Reply:
I thought it was better style because you don't have to change the test in case you want to test a different season and year.

Reply:
if you want to parametrize the test the correct style is to use pytest.parametrize

Reply:
I will look that up and do it. Don't merge the pr yet